sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubProducerClient.cs

```diff
@@ -46,8 +46,6 @@ namespace Azure.Messaging.EventHubs.Producer
     ///   </para>
     /// </remarks>
     ///
-    /// <seealso cref="EventHubBufferedProducerClient" />
```

Comment:
Need to wait until we're ready to mark the buffered producer as `public`;   an issue will be filed to align the comments and discuss trade-offs similar to the updates to the buffered producer in this change set.